client/src/Component/MainTable.js

```diff
@@ -26,11 +26,18 @@ const MainTable = (props) => {
 					<span className="visually-hidden">Loading...</span>
 				</div>
 			</div>
-		);
-	}
+		  );
+	  }
 	if (props.error != null) {
 		return <Alert variant="danger">{props.error}</Alert>;
-	}
+	  }
+
+	const validDate = (value) => {
```

Comment:
```suggestion
	const formatDate = (value) => {
```
Functions perform actions and usually start with a verb, so we could rename it accordingly